test/unittests/CMakeLists.txt

```diff
@@ -203,6 +171,38 @@ SET (SOURCES
     rest/restReply_test.cpp
     rest/RestService_test.cpp
     rest/rest_test.cpp
+
+    # serviceRoutines/badVerbGetOnly_test.cpp
```

Comment:
Leftover?

Reply:
I undesrtand the test has been disabled maybe due to the effort it would take to adjust them. In the past you mentioned they should be removed... If so, let's do that in this PR, no need to wait any longer.

Reply:
no, i decided to leave all those.
might do the effort to put all these unit tests back in, soon.
just not right now

Reply:
NTC